.shell-methods

```diff
@@ -88,3 +88,12 @@ function addAlias {
     # less defaults
     alias less='less -FSRX'
 }
+
+function addDockerAlias {
```

Comment:
@Cyb10101 is this function executable from within Bash/Zsh?

Reply:
yes